tests/test_pdf.py

```diff
@@ -29,6 +29,60 @@ def test_pdf_inputs(backend):
     )
 
 
+def test_invalid_pdf_inputs():
```

Comment:
A better, smaller test should be added. Just throwing this in for now. Suggestions welcome.

Reply:
can also use `pyhf.simplemodels`